### PR TITLE
(Legacy) Speed

### PR DESCRIPTION
**Pull request by NCPUser**
Not fasted speed but flags works on cheatmine (bad server) and mineblaze

### CODE REVIEW

**Comment by CzechHek** [comment]
You have to put that intave strafe value into Speed module itself and import it to the speed mode (otherwise it won't be visible, editable). Make sure to also only make it supported if intave speed mode is selected.

**Comment by NCPUser** [comment]
deleted oldintave strafe for useless

**Comment by ghost** [comment]
If this bypass does not work on Intave but on a specific server, you're better off renaming the mode to the server name that it bypasses on.

**Comment by NCPUser** [comment]
> okay

**Comment by NCPUser** [comment]
Renamed speed

**Comment by ghost** [comment]
I would recommend that you use an IDE.

**Comment by NCPUser** [comment]
> I would recommend that you use an IDE.

ok

**Comment by NCPUser** [comment]
fixed

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt [hunk 1]
```diff
@@ -10,11 +10,11 @@ import net.ccbluex.liquidbounce.event.MoveEvent
...
 import net.ccbluex.liquidbounce.utils.MovementUtils.speed
 
-object MineBlazeSpeed : SpeedMode("MineBlazeHop") {
+object MineBlazeHop : SpeedMode("MineBlazeHop") {
            override fun onMotion() {
              if (mc.thePlayer.motionY > 0.003) {
-                mc.thePlayer.motionX *= 1.0015
-                mc.thePlayer.motionZ *= 1.0015
+                mc.thePlayer.motionX = 1.0015
+                mc.thePlayer.motionZ = 1.0015
```
Comment (CzechHek) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: ?

Comment (NCPUser) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: motion speed

Comment (CzechHek) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: look at it

Comment (NCPUser) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: motionZ and motionX

Comment (CzechHek) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: look at the assignment operator?

Comment (FDPUser443) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: "*=" maybe ? =))

Comment (NCPUser) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt]: ok fixed soon

**Comment by ghost** [comment]
> You have to put that intave strafe value into Speed module itself and import it to the speed mode (otherwise it won't be visible, editable). Make sure to also only make it supported if intave speed mode is selected.

^^^^^^^ @NCPUser

**Comment by CzechHek** [comment]
The strafe value was removed but there is a different issue, he didn't add the mode to speed modes list.

**Comment by NCPUser** [comment]
added speed on speed.kt @mems01

**Comment by ghost** [comment]
Tell me when to merge.

**Comment by NCPUser** [comment]
now merge @mems01

**Comment by FDPUser443** [comment]
@NCPUser, It doesn't jump automatically when I'm moving. Is it OK for this speed mode?

**Comment by FDPUser443** [comment]
The timer speed before the first jump is 1.0f but after performing this one it remains 1.06f, even though I stand still and don't move anymore. Is it OK too or not?

**Pull request merged**